cmd/orchestrators.go

```diff
@@ -35,12 +36,13 @@ func newOrchestratorsCmd() *cobra.Command {
 	f := command.Flags()
 	f.StringVar(&oc.orchestrator, "orchestrator", "", "orchestrator name (optional) ")
 	f.StringVar(&oc.version, "version", "", "orchestrator version (optional)")
+	f.BoolVar(&oc.windows, "windows", false, "orchestrator platform (optional)")
```

Comment:
Could you add a note here that the windows flag applies to the Kubernetes orchestrator only?

For example: `orchestrator platform (optional, applies to Kubernetes only)`